### PR TITLE
HH-2: updating gitignore

### PR DESCRIPTION
**Pull request by jim-spa**

### CODE REVIEW

**Comment by jim-spa** [comment]
Adding ignore for .idea folder

File: .gitignore [hunk 1]
```diff
@@ -34,7 +34,9 @@ captures/
 
 # Intellij
 *.iml
-.idea/workspace.xml
+.idea
 
 # Keystore files
 *.jks
+.idea
```
Comment (M-spa) [inline .gitignore]: Why is ".idea" on here twice?

Comment (jim-spa) [reply .gitignore]: oops! I'll fix it..

**Review by M-spa — APPROVED** [review]
Looks good

**Pull request merged**